task/monorail-server/src/server.rs

```diff
@@ -365,9 +366,7 @@ impl<'a, R: Vsc7448Rw> idl::InOrderMonorailImpl for ServerImpl<'a, R> {
                 if let Some(Err(e)) = self.bsp.phy_fn(port, |phy| {
                     phy.read(phy::STANDARD::INTERRUPT_STATUS())
                 }) {
-                    return Err(e)
-                        .map_err(MonorailError::from)
-                        .map_err(RequestError::from);
+                    return Err(RequestError::from(MonorailError::from(e)));
```

Comment:
how dare you, clippy

Reply:
Clippy says to meet it behind the school at 3pm and, quote, "let's finish this"